server/src/items/classes/spell_scrolls/SpellScroll.js

```diff
@@ -0,0 +1,48 @@
+const Item = require("../Item");
+
+class SpellScroll extends Item {
+    onUsed() {
+        console.log("SpellScroll used")
+        super.onUsed();
+    }
+    
+    getBoardTilesInRange(inputRange, fnApplyToBoardTile) {
+        const { 
+                row,
+                col,
+                board,
+            } = this.owner;
+
+        const range = this.validateRange(inputRange);
+        const rangePlusOne = range + 1        
+        let rowOffset;
+        let colOffset;        
+        let boardTile;
+        let tiles = [];
+       
+        for (rowOffset = -range; rowOffset < rangePlusOne; rowOffset += 1) {
+            for (colOffset = -range; colOffset < rangePlusOne; colOffset += 1) {
+                // Check row is valid.
+                if (board.grid[row + rowOffset] === undefined) continue;
+                boardTile = board.grid[row + rowOffset][col + colOffset];
+                // Check col is valid.
+                if (boardTile === undefined) continue;
+                tiles.push(boardTile);
+            }
+        }
+        return tiles;
+    }
```

Comment:
This method seems like it would be more suitable as a method on the Board class (`getTilesInRange`?), as there are several other similar board utilities like this already there, and this method may be used in the future for things other than spell scrolls.
So it can be used like `this.owner.board.getTilesInRange(6);`

Reply:
what would you say to add method
`board.getTilesInEntityRange(e:Destroyable, n:Number)` to board? So it can be used like you suggested
`this.owner.board.getTilesInEntityRange(this.owner, 6)`